scalardl-test/src/main/java/scalardl/transfer/TransferChecker.java

```diff
@@ -44,8 +44,9 @@ public void execute() {
 
   private List<JsonObject> readBalancesWithRetry() {
     logInfo("reading latest assets...");
-
-    Retry retry = Common.getRetryWithExponentialBackoff("readBalances");
+    int maxRetry = (int)config.getUserLong("test_config","checker_max_retries_for_read", 10L);
```

Comment:
Please follow [Google style](https://google.github.io/styleguide/javaguide.html):
```suggestion
    int maxRetry = (int)config.getUserLong("test_config", "checker_max_retries_for_read", 10L);
```

Reply:
Thank you for the suggestion.
I will ensure google style coding standards is followed.